migrations/2022-04-04-183652_update_community_aggregates_on_soft_delete/up.sql

```diff
@@ -0,0 +1,64 @@
+drop trigger if exists community_aggregates_post_count on post;
+drop trigger if exists community_aggregates_comment_count on comment;
+
+create or replace function community_aggregates_post_count()
+    returns trigger language plpgsql
+as $$
+begin
+  IF (TG_OP = 'INSERT') THEN
+update community_aggregates
+set posts = posts + 1 where community_id = NEW.community_id;
+ELSIF (TG_OP = 'DELETE'  OR (TG_OP = 'UPDATE' AND OLD.deleted = 'f' AND NEW.deleted = 't')) THEN
```

Comment:
Unfortunately besides the deleted column, there is also the removed column. (deleted is when the creator deletes it, removed is when a mod or admin removes it).